cmd/helm/installer/options.go

```diff
@@ -46,6 +46,9 @@ type Options struct {
 	// ServiceAccount is the Kubernetes service account to add to Tiller.
 	ServiceAccount string
 
+	// Force allows to force upgrading tiller if deployed version is greater than current version
+	ForceUpgrade bool
```

Comment:
I would prefer a different name, since `helm init --force` could be ambiguous. `--force-upgrade` sounds fine, though. (or `--allow-downgrade` or `--with-downgrade` or something)